sphinx/writers/latex.py

```diff
@@ -253,21 +253,21 @@ def rstdim_to_latexdim(width_str: str, scale: int = 100) -> str:
     if scale == 100:
         float(amount)  # validate amount is float
         if unit in ('', "px"):
-            res = "%s\\sphinxpxdimen" % amount
+            res = r"%s\sphinxpxdimen" % amount
         elif unit == 'pt':
-            res = '%sbp' % amount  # convert to 'bp'
+            res = r"%sbp" % amount  # convert to 'bp'
```

Comment:
is this done for style uniformity? (or as by-product of big regex search/replace...)

Reply:
Sorry, this should be discussed in another PR. Reverted.